src/main/java/frc/team2412/robot/subsystems/ArmSubsystem.java

```diff
@@ -396,11 +396,11 @@ public double calculateArmPID() {
 	 */
 	public double calculateArmFeedforward() {
 		// -0.111639604	0.064541504
-		double shoulderPosition = getShoulderPosition();
+		double wristPosition = getWristPosition();
 
-		return ARM_K_S * Math.signum(armGoal - shoulderPosition) + shoulderPosition <= 0.48
+		return ARM_K_S * Math.signum(armGoal - wristPosition) + wristPosition <= 0.48
```

Comment:
I'm not sure why we're reading from wrist here instead of elbow, but it does work, so...